pkg/vulnsrc/vulnerability/vulnerability.go

```diff
@@ -14,16 +14,21 @@ import (
 )
 
 const (
-	trivyIgnore = ".trivyignore"
+	defaultTrivyIgnore = ".trivyignore"
```

Comment:
How about exporting this const?
```suggestion
	DefaultIgnoreFile = ".trivyignore"
```